browser/brave_browser_process.cc

```diff
@@ -0,0 +1,161 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
I originally did that, but changed it at @iefremov's request - perhaps he can chime in here

Reply:
yea, this seems a bit odd to me as well and upstream browser_process is mostly pure virtual.

Reply:
ok, consensus seems to be to follow upstream - fixed

Reply:
My reasons were:
1. Every time one adds a new member to brave browser process, they would forget to add a corresponding stub to testing browser process, fail tests compilation, and then fix that (instead of simply adding a stub to the base class) 
2. It would be clear that the base class does not contain any useful logic, while a testing class has only meaningful methods. Now we have a mix in the testing class

Reply:
as I mentioned to @antonok-edm in DM I don't have a strong opinion one way or the other on this so I left it to him. I don't think we should block on this, especially since @antonok-edm will be on PTO